packages/contracts/contracts/EpochKeyLiteProofVerifier.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IUnirep} from './interfaces/IUnirep.sol';
+import {IVerifier} from './interfaces/IVerifier.sol';
+
+import {EpochKeyProofVerifier} from './EpochKeyProofVerifier.sol';
+
+contract EpochKeyLiteProofVerifier is IUnirep {
+    IVerifier public epochKeyLiteVerifier;
+    EpochKeyProofVerifier public epochKeyProofVerifier;
```

Comment:
In Solidity there's a [large-ish cost](https://eips.ethereum.org/EIPS/eip-2929#specification) to accessing other contracts in a call. Instead of storing a reference to the `EpochKeyProofVerifier` it would be better to define a base class that implements `decodeEpochKeyControl`, and then have both `EpochKeyProofVerifier` and `EpochKeyLiteProofVerifier` inherit the base contract. Then both instances have their own copy of `decodeEpochKeyControl`.